source/dotnet/wholesale-api/Orchestrations/Functions/Calculation/Activities/UpdateCalculationStatusActivity.cs

```diff
@@ -54,45 +70,24 @@ internal class UpdateCalculationStatusActivity(
 
             // If state wasn't changed, do nothing
             if (calculation.OrchestrationState == newState)
-                return;
+            {
+                _logger.LogInformation(
+                    "Do not set calculation state since it didn't update. Current state: {CurrentOrchestrationState}, calculation id: {CalculationId}",
```

Comment:
```suggestion
                    "Did not update calculation state since it didn't change. Current state: {CurrentOrchestrationState}, calculation id: {CalculationId}",
```
Would this make sence?